docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -0,0 +1,144 @@
+---
+product: date-pickers
+title: Date and Time Picker - Base concepts
+packageName: '@mui/x-date-pickers'
+githubLabel: 'component: pickers'
+materialDesign: https://m2.material.io/components/date-pickers
+waiAria: https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/datepicker-dialog.html
+---
+
+# Date and Time Pickers - Base concepts
+
+<p class="description">The Date and Time pickers expose a lot of components to fit your every needs.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+## Basic usage
+
+All the components have a `value` / `onChange` API to set and control the values:
+
+{{"demo": "ControlledComponent.js"}}
+
+## Imports
+
+All the components exported by `@mui/x-date-pickers` are also exported by `@mui/x-date-pickers-pro` but without the nested imports.
+
+For example, to use the `DatePicker` component, the following three imports are valid:
+
+```tsx
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { DatePicker } from '@mui/x-date-pickers';
+import { DatePicker } from '@mui/x-date-pickers-pro';
+```
+
+## Other components
+
+### Choose interaction style
+
+Depending on your use case, different interaction styles are preferred.
+
+- For input editing with a popover for mouse interaction, use the _Picker_ components:
+
+{{"demo": "BasicDatePicker.js", "hideToolbar": true, "bg": "inline"}}
+
+- For input-only editing, use the _Field_ components:
+
+{{"demo": "BasicDateField.js", "hideToolbar": true, "bg": "inline"}}
+
+- For inline editing, use the _Calendar / Clock_ components:
+
+{{"demo": "BasicDateCalendar.js", "hideToolbar": true, "bg": "inline"}}
+
+:::info
+Each _Picker_ is a combination of one _Field_ and one or several _Calendar / Clock_ components.
+For example, the `DatePicker` is the combination of the `DateField` and the `DateCalendar`.
+
+The _Calendar / Clock_ components are rendered inside a _Popover_ on desktop and inside a _Modal_ on mobile.
+:::
+
+### Date or time editing ?
+
+The Date and Time Pickers are divided into six families of components.
+The demo below shows each one of them using their field component:
+
+{{"demo": "ComponentFamilies.js"}}
+
+### Responsiveness
+
+Each _Picker_ is available in a desktop, mobile and responsive variant:
+
+- The responsive component (e.g. `DatePicker`) which renders the desktop component or the mobile one depending on the device it runs on.
+
+- The desktop component (e.g. `DesktopDatePicker`) which works best for mouse devices and large screens.
+  It renders the views inside a popover and allows editing values directly inside the field.
+
+- The mobile component (e.g. `MobileDatePicker`) which works best for touch devices and small screens.
+  It renders the view inside a modal and does not allow editing values directly inside the field.
```

Comment:
Maybe put the responsive variant description at the bottom to align with the order described above? 🤔 
```suggestion
- The desktop component (e.g. `DesktopDatePicker`) which works best for mouse devices and large screens.
  It renders the views inside a popover and allows editing values directly inside the field.

- The mobile component (e.g. `MobileDatePicker`) which works best for touch devices and small screens.
  It renders the view inside a modal and does not allow editing values directly inside the field.

- The responsive component (e.g. `DatePicker`) which renders the desktop component or the mobile one depending on the device it runs on.
```

Reply:
Jose asked to have the responsive at the top, I'll fix the order above instead

Reply:
Sounds like a plan as well. As long as we are staying consistent 👍